### ISSUE TITLE
Using the SDK with bitbox-gui

### ISSUE THREAD

**Issue by sambengtson** [Bitcoin-com/bitbox-javascript-sdk#10]
Hello,

I'm using the default scaffolding project for NodeJS trying to use the SDK with bitbox-gui.  I'm declaring BITBOX as such:
```

let BITBOXCli = require('bitbox-cli/lib/bitbox-cli').default;
let BITBOX = new BITBOXCli({
  protocol: "http",
  host: "localhost",
  port: "8332",
  username: "",
  password: ""
});
```

It doesn't seem that the SDK is using my local node.  I'm basing this on calling 

> BITBOX.Address.details

with my coinbase address.  It always resolves with a balance of 0 regardless if I am running the bitbox-gui or not.  Am I thinking about this wrong?  Appreciate the work you guys / gals have done.

**Comment by cgcardona** [comment]
Hi. Thanks for checking out BITBOX. This API is out of date

```js
let BITBOXCli = require('bitbox-cli/lib/bitbox-cli').default;
let BITBOX = new BITBOXCli({
  protocol: "http",
  host: "localhost",
  port: "8332",
  username: "",
  password: ""
});
```
Can you point me to where you saw it so I can remove/update it? Thx

The short answer is what you're asking isn't currently possible but is on our roadmap.

We're working to make our GUI be a testnet wallet and single click blockchain which supports 100% of the https://rest.bitcoin.com/ API but we're not there yet. For now you can use the following API

```js
let BITBOXCli = require('bitbox-cli/lib/bitbox-cli').default;
let BITBOX = new BITBOXCli();
```
Or for testnet:
```js
let BITBOXCli = require('bitbox-cli/lib/bitbox-cli').default;
let BITBOX = new BITBOXCli({
  restURL: "https://trest.bitcoin.com/v1/"
});
```

**Comment by cgcardona** [comment]
And the further answer your question the balance that you see in GUI won't exist on the actual blockchain. That value only exists locally in GUI for testing your scripts with.

ATM GUI still needs quite a bit of work and we're hoping to dedicate an engineer to working on it full time soon.

**Comment by sambengtson** [comment]
Thanks @cgcardona ,

I've looked at so many resources, I can't remember exactly where I saw it.  I'm pretty sure it was an old, resolved, github issue.  If I find it again I will let you know.

Is there a better place for me to ask general questions like this?  I would rather not create Github issues for implementation questions.  Got a rather large project going that I would like to use Bitbox for
